docs/guides/deploy.md

```diff
@@ -88,25 +88,25 @@ If your project requires variables, they can be added in project settings.
 
 > Changes to variables will require a redeploy, which can be triggered from the project's home.
 
-<img alt="...." src="../../assets/img/guides/deploy/new-environment-variable.png" height="337.5" />
+<img alt="Add environment variables for production and staging" src="../../assets/img/guides/deploy/new-environment-variable.png" height="337.5" />
 
 <br/>
 
 ## Create an environment
 
 An environment represents a deployed instance of your application in a specific AWS account and region. You can have one or more environments based on the development workflow you follow (e.g. dev, staging, prod), or if you need environments in other regions or using different credentials.
 
-> Once you've created an environment, Nitric Deploy will automatically initiate your first deployment to the cloud.
+> Once you've created an environment, Nitric Deploy will automatically initiate your first cloud deployment.
 
-<img alt="...." src="../../assets/img/guides/deploy/new-environment.png" height="337.5" />
+<img alt="A project in Nitric Deploy with no environments created" src="../../assets/img/guides/deploy/new-environment.png" height="337.5" />
```

Comment:
now that we are moving to stacks instead of enviroments, should we update here too?